habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/ValidatePythonCodeMojo.java

```diff
@@ -0,0 +1,135 @@
+package org.technologybrewery.habushu;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugins.annotations.LifecyclePhase;
+import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.plugins.annotations.Parameter;
+import org.apache.maven.plugins.annotations.ResolutionScope;
+import org.technologybrewery.habushu.exec.PoetryCommandHelper;
+
+/**
+ * Leverages the lint package to validate both source and test Python
+ * directories using Poetry's run command.
+ */
+@Mojo(name = "validate-python-code", defaultPhase = LifecyclePhase.PROCESS_CLASSES, requiresDependencyResolution = ResolutionScope.COMPILE)
```

Comment:
A: Might make more sense for this to be in the `processSources` phase by default. I'm thinking if we add python compilation, that would be attached to the `compile` phase which happens _before_ process classes.  Logically it makes more sense to lint before you compile.